lightning/src/offers/merkle.rs

```diff
@@ -318,6 +338,25 @@ mod tests {
 		);
 	}
 
+        #[test]
+        fn compute_tagged_hash() {
+                let unsigned_invoice_request = OfferBuilder::new("foo".into(), recipient_pubkey())
+                        .amount_msats(1000)
+                        .build().unwrap()
+                        .request_invoice(vec![1; 32], payer_pubkey()).unwrap()
+                        .payer_note("bar".into())
+                        .build().unwrap();
+
+                // Simply test that we can grab the tag and merkle root exposed by the accessor
+                // functions, then use them to succesfully compute a tagged hash.
+                let taggedhash = unsigned_invoice_request.as_ref();
```

Comment:
Let's not import `tagged_hash` and qualify it with `super::` below so that this can be named with an underscore.